### PR TITLE
Added SequenceRNAStringSetTrack class

### PR DESCRIPTION
**Pull request by FelixErnst**
Refactored some internals to support SequenceTrack object representing other nucleotide sequences than just DNA. For usage case see RNAmodR package, which uses additional XStringSet classes from the Modstrings package.

### CODE REVIEW

**Comment by FelixErnst** [comment]
Hi @ivanek 

just wanted bump this. Any thoughts on this?

Felix

**Comment by FelixErnst** [comment]
Hi @ivanek 
any news on this and the other PR https://github.com/ivanek/Gviz/pull/22?

**Comment by ivanek** [comment]
Hi,
I had a look on it, but the it cannot be merged as it is, it breaks the default coloring of `SequenceTrack`s.

**Comment by FelixErnst** [comment]
I am not sure what you mean. Would you be able to give me an example?

File: R/Gviz.R [hunk 1]
```diff
@@ -1185,7 +1185,6 @@
                                                  cex=1,
                                                  col="darkgray",
                                                  complement=FALSE,
-                                                 fontcolor=getBioColor("DNA_BASES_N"),
```
Comment (ivanek) [inline R/Gviz.R]: You need to set `fontcolor` to `NULL` to get the coloring work.

**Comment by ivanek** [comment]
Have you checked the resulting vignette? It does not plot the sequences.
Instead of deleting the the line in the Gviz.R it would be better to set the fontcolor=NULL.  
I will merge it and do the mod + edit of documentation.

**Comment by FelixErnst** [comment]
Well I thought I did. So that is a bit confusing to me right now. Thanks for fixing this.

**Pull request merged**